modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/messaging/CheckRedirectNotFoundEntriesMessageListener.java

```diff
@@ -47,13 +50,21 @@
  */
 @Component(
 	configurationPid = "com.liferay.redirect.internal.configuration.RedirectConfiguration",
-	service = MessageListener.class
+	service = {}
 )
 public class CheckRedirectNotFoundEntriesMessageListener
 	extends BaseMessageListener {
 
 	@Activate
-	protected void activate(Map<String, Object> properties) {
+	protected void activate(
```

Comment:
@yuhai You should never modify your production code just because of test need it.